src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC, memo } from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+
+import { sortTypes } from './../../constants/constants';
+
+import { PersonLink, SearchLink } from '../';
+
+interface Props {
+  people: Person[];
+  sort: string | null;
+  order: string | null;
+}
+export const PeopleTable: FC<Props> = memo(({ people, sort, order }) => {
+  const { slug } = useParams();
+
+  const getSortParams = (sortBy: string) => {
+    if (sortBy === sort && !order) {
+      return { sort: sortBy, order: 'desk' };
+    }
+
+    if (sortBy !== sort && !order) {
+      return { sort: sortBy, order: null };
+    }
+
+    return { sort: null, order: null };
```

Comment:
There seems to be a potential issue in the `getSortParams` function. The logic for determining the `order` when `sortBy` is equal to `sort` and `order` is not set might not be correct. Consider reviewing the logic to ensure it correctly toggles between ascending and descending order.